prac_05/state_names.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+state = input("Enter short state: ")
+while state != "":
+    if state.upper() in STATE_NAMES:
```

Comment:
maybe put the .upper after the state = input incase you had to use it anywhere else